hw/ip/hmac/dv/cov/hmac_cov_unr_excl.el

```diff
@@ -0,0 +1,311 @@
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "2249151938" "(shaf_rready && txcnt_eq_1a0) 1 -1" (1 "01")
+ANNOTATION: "VC_COV_UNR"
+Condition 6 "4245900456" "(shaf_rready && ((!txcnt_eq_1a0))) 1 -1" (2 "10")
+CHECKSUM: "2480142477 1044153900"
+INSTANCE: tb.dut.u_hmac
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "1671356253" "(fifo_wsel && fifo_wvalid) 1 -1" (1 "01")
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "1671356253" "(fifo_wsel && fifo_wvalid) 1 -1" (2 "10")
+ANNOTATION: "VC_COV_UNR"
+Condition 16 "1320860652" "((sel_rdata == SelFifo) ? fifo_rdata : ('{(*adjust*)default:'0, (*adjust*)default:'0})) 1 -1" (1 "0")
+ANNOTATION: "VC_COV_UNR"
+Condition 19 "3519635535" "((sel_msglen == SelOPadMsg) ? ((BlockSize64 + 64'h0000000000000100)) : '0) 1 -1" (1 "0")
+CHECKSUM: "3777752306 3098758485"
+INSTANCE: tb.dut
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "2339369910" "(msg_write && sha_en && packer_ready) 1 -1" (2 "101")
+ANNOTATION: "VC_COV_UNR"
+Condition 6 "2180223097" "(((!reg_fifo_wvalid)) && ((!fifo_rvalid)) && hmac_core_idle && sha_core_idle) 1 -1" (1 "0111")
+CHECKSUM: "2024123546 4210417746"
+INSTANCE: tb.dut.u_tlul_adapter.u_rspfifo
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2331864568" "(gen_normal_fifo.full ? (1'(Depth)) : ((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value))))) 1 -1" (2 "1")
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "1926118060" "((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value)))) 1 -1" (1 "0")
+ANNOTATION: "VC_COV_UNR"
+Condition 3 "1937111844" "((gen_normal_fifo.fifo_empty && wvalid_i) ? wdata_i : gen_normal_fifo.storage_rdata) 1 -1" (2 "1")
+ANNOTATION: "VC_COV_UNR"
+Condition 4 "4208363759" "(gen_normal_fifo.fifo_empty && wvalid_i) 1 -1" (1 "01")
+ANNOTATION: "VC_COV_UNR"
+Condition 4 "4208363759" "(gen_normal_fifo.fifo_empty && wvalid_i) 1 -1" (3 "11")
+ANNOTATION: "VC_COV_UNR"
+Condition 5 "2860893437" "(gen_normal_fifo.empty ? 'b0 : gen_normal_fifo.rdata_int) 1 -1" (1 "0")
+CHECKSUM: "2024123546 3257640859"
+INSTANCE: tb.dut.u_tlul_adapter.u_reqfifo
+ANNOTATION: "VC_COV_UNR"
+Condition 2 "1926118060" "((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value)))) 1 -1" (1 "0")
+CHECKSUM: "2024123546 95776463"
+INSTANCE: tb.dut.u_tlul_adapter.u_sramreqfifo
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2331864568" "(gen_normal_fifo.full ? (1'(Depth)) : ((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value))))) 1 -1" (2 "1")
```

Comment:
Excluded code: https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/rtl/prim_fifo_sync.sv#L83
hmac msg fifo u_sramreqfifo never full because it is WO.